experiments/fiction_experiments.hpp

```diff
@@ -49,15 +49,60 @@ inline constexpr uint64_t xor5_maj         = 0b000000000000000000000000000000000
 inline constexpr uint64_t parity           = 0b0000000000000000000000000000000000000000010000000000000000000000;
 inline constexpr uint64_t fontes18         = 0b0000000000000000000000000000000000000000011111111111111110000000;
 
-inline constexpr uint64_t all              = 0b0000000000000000000000000000000000000000011111111111111111111111;
+/* ISCAS benchmarks */
+inline constexpr uint64_t c17_i            = 0b0000000000000000000000000000000000000000100000000000000000000000;
```

Comment:
I'd rather have this `c17` and Fontes' one something like `c17_f` because it originates from the ISCAS benchmark set